module Effectful.Exception (throwString) where

import Control.Exception.Safe
